packages/node_modules/brightspace-auth-keys-dynamodb-store/test/public-key-store.spec.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const
+	DynamoPublicKeyStore = require('../src/public-key-store'),
+	sinon = require('sinon'),
+	chai = require('chai'),
+	assert = require('chai').assert,
+	DynamoDbLocal = require('dynamodb-local'),
+	AWS = require('aws-sdk'),
+	chaiAsPromised = require('chai-as-promised'),
+	clock = require('../src/clock');
+
+chai.use(chaiAsPromised);
+
+describe('public-key-storeintegration tests', function() {
+	const expectedPublicKey = JSON.stringify({
+		kid: 'TestPublicKey'
+	});
+
+	const dynamoLocalPort = 8000;
+	const expiryTime = clock() + 10000;
+	const oldExpiryTime = clock() - 10000;
+	const tableName = 'auth_public_keys';
+
+	let provisioner;
+	let db;
+
+	before(() => {
+		return DynamoDbLocal.launch(dynamoLocalPort, null, ['-sharedDb']).then(() => {
+			db = new AWS.DynamoDB({
```

Comment:
Any way to clear the table between tests?

Reply:
Not that I could find. Tests should generally clean up after themselves.

Reply:
just `.stop()` and then start a new one?

Reply:
Possible, but heavy-handed and may slow down test execution noticeably. Similarly it's possible to call deleteTable followed by createTable, but other than writing fewer lines of code there doesn't seem to be a good reason to take a heavy handed approach between tests instead of expecting the tests to remove the data they add.

Reply:
I decided to just do it since it's what we're doing elsewhere